### ISSUE TITLE
404 not Found

### ISSUE THREAD

**Issue by Nrir** [stcr/subscribe-to-comments-reloaded#364]
Hi,

I have an issue with StCR.
The link to subscribe to comments under post leads to a 404 page.

All was fine until 27th july, no update of StCR nor WP.

Management page is the default virtual page : /comment-subscriptions/,  and permalinks are %postname%.

Can you help me ?

Thanks

**Comment by Reedyseth** [comment]
Hello @Nrir , I apologize for the late response. Have you check if this issue might be related to #366 ? 

So far I have tested the plugin and have no issue on the manager page.

**Comment by Nrir** [comment]
Hi, sorry for not having replied before ! I'm not so much late, I am ? ... ;)

It's not the same issue as the one you linked me, we don't use this plugin.

I solved this by created a real management page, and it's working fine.

Before that, I've tried to disable every other plugin, reset StCR, etc ... without any improvement.

Though, I got no problems with real page and that's the most important right now ;)

Thanks for your answer
